lib/owl_plot.ml

```diff
@@ -1593,14 +1593,24 @@ let scatterhist = None
 
 (* other plots *)
 
+let _rotate270_ x =
+  let w, h = Owl_dense_matrix.D.shape x in
```

Comment:
OK, I see. Then probably you need `transpose` + `slice`. I think the better solution is I implement `rotate` function in the core. I will merge this PR then replace your "_rotate_" function later.